packages/camera/camera_windows/windows/camera.cpp

```diff
@@ -250,7 +283,8 @@ void CameraImpl::OnCaptureError(const std::string& error) {
     channel->InvokeMethod(kErrorEvent, std::move(message_data));
   }
 
-  SendErrorForPendingResults("capture_error", error);
```

Comment:
This is one of the breaking changes. This case can be triggered if I unplug my camera while the preview us running. It's not clear why this case had its own error code, it seems better for these error codes to be consistent.